src/App.tsx

```diff
@@ -45,7 +45,49 @@ export const App = () => {
   const { width, ref } = useResizeDetector<HTMLDivElement>();
   const isSmallScreen = width !== undefined && width < 768;
 
-  const { host, qc, setHostAndQc } = useExploreUrl() ?? {};
+  const { host, qc, setHostAndQc, dateSubmitted } = useExploreUrl() ?? {};
+
+  const formatDateSubmittedAsString = () => {
+    if (dateSubmitted?.dateSubmitted) {
+      if (dateSubmitted.dateSubmitted === 'AllTimes') {
+        return null;
+      }
+
+      let dateString: string = '';
+
+      switch (dateSubmitted.dateSubmitted) {
```

Comment:
Please reuse the [existing function](https://github.com/cevo-public/cov-spectrum-website/blob/3c18131ead880099d4ae3344f80bd58003328466/src/data/DateRangeSelector.ts#L129)